webmails/snappymail/config/nginx-snappymail.conf

```diff
@@ -38,4 +47,9 @@ server {
     location ^~ /data {
       deny all;
     }
+
+    location /health {
+        add_header Content-Type text/plain;
+        return 200;
```

Comment:
I wonder if it's worth making it a PHP script to ensure that FPM is fine too.

Reply:
This would be good. You don't need a script though, you can use `ping.path` and `ping.response` of the fpm daemon.

Reply:
Yes. With this we test both nginx and fpm-daemon. I will enable this and modify the health check in the Dockerfile.